src/chart/funnel/funnelLayout.js

```diff
@@ -71,9 +71,19 @@ function labelLayout(data) {
         var linePoints;
 
         if (isLabelInside) {
-            textX = (points[0][0] + points[1][0] + points[2][0] + points[3][0]) / 4;
-            textY = (points[0][1] + points[1][1] + points[2][1] + points[3][1]) / 4;
-            textAlign = 'center';
+            if (labelPosition === 'insideLeft'){
```

Comment:
Add space: `if (labelPosition === 'insideLeft') {`.
Also change the similar problems.